admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -446,27 +446,29 @@ var featuresCollection = (function() {
  * Suppliers management
  */
 var supplier = (function() {
-  var defaultSupplierRow = $('#default_supplier_list');
-  var isInit = false;
+
+  var supplierInputManage = function(input) {
+    var supplierDefaultInput = $('#form_step6_suppliers input[name="form[step6][default_supplier]"][value=' + $(input).val() +']');
+    if($(input).is(':checked')) {
+      $(supplierDefaultInput).prop('disabled', false).show();
```

Comment:
`supplierDefaultInput` is a jQuery element so there is no need to wrap it again with $().